General/InterestingsToKnowForBeginners.md

```diff
@@ -67,6 +69,20 @@ The second way is shorter and often more readable.
 
 > TODO
 
+### Calypso (System browser)
+
+- Display the differences between two methods
+	* Click on a method
+	* Hold down the SHIFT-key
+	* Click on the second method
+	* Click on the tab Diff to see the differences
+	
+- Editing 2 methods side by side
```

Comment:
It works with as many methods as you want by the way :-)

Maybe it is worth to specify it.

Reply:
Done